lib/src/util/fuzzyFilter.ts

```diff
@@ -5,18 +5,20 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 import Fuse from 'fuse.js';
+import { map, prop } from 'ramda';
 
-export default function fuzzyFilter<T>(input: T[], term: string, options: Fuse.FuseOptions<T>): T[] {
+export default function fuzzyFilter<T>(input: T[], term: string, options: Fuse.IFuseOptions<T>): T[] {
   // fuse.js defaults "shouldSort" option to true, which is counter intuitive and contradicts their usage example
   // fix it by setting shouldSort to false by default
   const normalizedOptions = {
+    ...options,
```

Comment:
Is there a reason this moved?

Reply:
> Is there a reason this moved?

Yeah, I moved it so that it wouldn't end up overwriting the defaults below.

Reply:
> Yeah, I moved it so that it wouldn't end up overwriting the defaults below.

I think that was intentional, we want to allow overwriting defaults.

Reply:
> > Yeah, I moved it so that it wouldn't end up overwriting the defaults below.
> 
> I think that was intentional, we want to allow overwriting defaults.

Changed in f10386b29b7074ceaf74dd50a8df8d4087d1df48